app/services/message_sender.rb

```diff
@@ -8,11 +8,16 @@ def initialize(employee, message)
 
   def run
     configure_slack
-    channel_id = SlackChannelIdFinder.new(employee.slack_username, client).run
 
-    if channel_id
+    if employee.slack_channel_id.nil?
```

Comment:
@jessieay no problem -- I just committed a change that adds a few tests to MessageSender. I also refactored the sweet to be more readable. Let me know what you think. Happy to adjust my work as needed.